src/components/table/table.stories.js

```diff
@@ -0,0 +1,118 @@
+  action('change')(e, tableOptions);
+};
+
+const buildRows = (pageSizeFromKnobs) => {
+  const pageSize = pageSizeFromKnobs;
+  const currentPage = store.get('currentPage');
+
+  const endIndex = pageSize * currentPage;
+  const startIndex = endIndex - pageSize;
+  const rowsCountries = countriesList.slice(startIndex, endIndex).toJS();
+
+  const rows = [
+    <TableRow key='header' as='header'>
+      <TableHeader
+        sortable name='name'
+        scope='col'
+      >
+        Country
+      </TableHeader>
+
+      <TableHeader scope='col'>Code</TableHeader>
+    </TableRow>
+  ];
+
+  if (store.get('sortOrder') === 'desc') {
+    rowsCountries.reverse();
+  }
+
+  rowsCountries.map(({ id, name, value }) => rows.push(
+    <TableRow key={ id } uniqueID={ id }>
+      <TableCell>{name}</TableCell>
+      <TableCell>{value}</TableCell>
+    </TableRow>
+  ));
+
+  return rows;
+};
+
+storiesOf('Table', module).add(
+  'default',
+  () => {
+    const pageSize = text('pageSize', '5');
+    const selectable = boolean('selectable', false);
+    const highlightable = boolean('hightTable', false);
+    const shrink = boolean('shrink', false);
+    const caption = text('Caption', 'Country and Country Codes');
+    const totalRecords = text('totalRecords', '50');
+    const paginate = boolean('paginate', false);
+    const showPageSizeSelection = paginate && boolean('showPageSizeSelection', false);
+    const theme = select('theme', OptionsHelper.themesBinary, Table.defaultProps.theme);
+
+    return (
+      <State store={ store }>
+        {state => (
+          <Table
+            actionToolbarChildren={ (context) => {
+              return [
+                <Button disabled={ context.disabled } key='single-action'>
+                  Test Action
+                </Button>,
+                <MultiActionButton
+                  text='Actions' disabled={ context.disabled }
+                  key='multi-actions'
+                >
+                  <Button>foo</Button>
+                  <Button>bar</Button>
+                  <Button>qux</Button>
+                </MultiActionButton>
+              ];
+            } }
+            path='/countries'
+            sortOrder={ state.sortOrder }
+            sortedColumn={ state.sortedColumn }
+            caption={ caption }
+            currentPage={ state.currentPage }
+            shrink={ shrink }
+            highlightable={ highlightable }
+            pageSize={ pageSize }
+            selectable={ selectable }
+            paginate={ paginate }
+            actions={ { delete: { icon: 'bin' }, settings: { icon: 'settings' } } }
+            totalRecords={ totalRecords }
+            showPageSizeSelection={ showPageSizeSelection }
+            onChange={ handleChange }
+            theme={ theme }
+          >
+            {buildRows(pageSize)}
+          </Table>
+        )}
+      </State>
+    );
+  },
+  { notes: { markdown: notes } }
+);
```

Comment:
Table Component markup not appearing in Story Source:

![Screenshot 2019-03-13 at 15 23 39](https://user-images.githubusercontent.com/4471057/54291190-20472f80-45a4-11e9-8dbc-2617422cc02d.png)